integration_tests/run_pyspark_from_build.sh

```diff
@@ -129,7 +129,7 @@ else
         # below where the processes are launched.
         GPU_MEM_PARALLEL=`nvidia-smi --query-gpu=memory.free --format=csv,noheader | awk '{if (MAX < $1){ MAX = $1}} END {print int((MAX - 2 * 1024) / ((1.5 * 1024) + 750))}'`
         CPU_CORES=`nproc`
-        HOST_MEM_PARALLEL=`cat /proc/meminfo | grep MemAvailable | awk '{print int($2 / (5 * 1024 * 1024))}'`
+        HOST_MEM_PARALLEL=`cat /proc/meminfo | grep MemAvailable | awk '{print int($2 / (5 * 1024 * 1536))}'`
```

Comment:
Can you please update the comments just above this as the amount is no longer 5 GiB?  Also the `5 * 1024 * 1024` was clear what it was, but `5 * 1025 * 1536` is much less clear that it is 7.5 GiB.  Could we try and rearrange that to make it more clear? `7.5 * 1024 * 1024` perhaps?

Reply:
sure, thanks for the suggestion